File: Assignment_27/program4.c
// Write a program which accept string from user and copy small characters of that string into another string.

///////////////////////////////////////////////////////////////////////////////
// File Name   :  program4.c
// Description :  Accept string from user and  copy small characters of that string into another string.
// Autor       :  Ankita Anil Patil
// Date        :  31/05/2025
///////////////////////////////////////////////////////////////////////////////



/*
    Input : "Marvellous Multi OS"

    Output : "arvellous multi"
*/

#include<stdio.h>

void StrCpySmall(char *src, char *dest)
{
    if(src ==NULL)
    {
        printf("Invalid input ");
        return -1;
    }
    while(*src != '\0')
    {
        if((*src >= 'a') && (*src <= 'z'))
        {
            *dest = *src;
             dest ++;
        }
        src++;
    }
    dest = '\0';
}
int main()
{
    char arr[30] = "Marvellous multi OS";
    char brr[30];

    StrCpySmall(arr,brr);

    printf("Small letters are : %s \n",brr);
    // arvellousmulti
    return 0;
}

File: Assignment_53/program4.java
// Write a java program which accept two array from user and copy 
// the contents of that array into another array and return new array.  
/*
    Input :  1 2 3 
             4 5 6  
    Output : 1 2 3 4 5 6
*/ 

 ///////////////////////////////////////////////////////////////////////////////
// File Name   :  program4.java
// Description :  Accept two arrays from user and copy the contents of 
//                that array into another array and return new array.         
// Autor       :  Ankita Anil Patil
// Date        :  13/09/2025
///////////////////////////////////////////////////////////////////////////////

import java.util.*;

class MyArray 
{ 
        public int[] CopyArr(int Arr1[], int Arr2[]) 
        {   
            int i = 0, j = 0;
            int newArr[] = new int[Arr1.length + Arr2.length];

            for(i = 0 ; i < Arr1.length; i++)
            {   
               newArr[i] = Arr1[i]; 
            }

            for(j = 0 ; j < Arr2.length; j++)
            {                
               newArr[i++] = Arr2[j]; 
            }

            return newArr;
        } 
} 
public class program4
    
{   
    public static void main(String[] args) {
        

    Scanner sobj = new Scanner(System.in);


    System.out.println("Array 1: ");
    System.out.println("Enter size of array");
    int iSize = sobj.nextInt();


    int[] Arr1 = new int[iSize];
    System.out.println("Enter array 1 elements : ");

    int i = 0;
    
    for(i = 0; i < Arr1.length; i++)
    { 
       Arr1[i] = sobj.nextInt();
    }

    System.out.println("Array 2: ");
    System.out.println("Enter size of array");
    int iSize2 = sobj.nextInt();


    int[] Arr2 = new int[iSize2];
    System.out.println("Enter array  2 elements : ");

    for(i = 0; i < Arr2.length; i++)
    {
        Arr2[i] = sobj.nextInt();
    }
    
    
    MyArray mobj = new MyArray();
    int newArray [] = mobj.CopyArr(Arr1, Arr2);

    for(i = 0; i < newArray.length; i++)
    {
        System.out.print(newArray[i] + " \t");
    }
    
    
}
}
